docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -536,6 +536,81 @@ fields:
     code: countries_list()
     default: ${ AL_DEFAULT_COUNTRY }
 ---
+generic object: ALIndividual
+id: any service address
+question: |
+  At what address will you serve ${ x }?
+fields:
+  - ${ x }'s service address is: x.service_address
+    datatype: object_radio
+    choices:
+      - x.address if defined(x.address.attr_name("address")) else None
+      - x.mailing_address if defined(x.mailing_address.attr_name("address")) else None
+    object labeler: |
+      lambda y: y.on_one_line()      
+    none of the above: |
+      Somewhere else
+    disable others: True      
```

Comment:
Is the show if on address and mailing address you used in #639, i.e.
```yml
   show if:
      code: |
        defined("x.address.address") and defined("x.mailing_address.address")
```
not needed here?

Reply:
No, if the only option is `None` Docassemble won't display the field.